spec/spec_helper.rb

```diff
@@ -4,10 +4,10 @@
 include Mercury::TestUtils
 
 # the block must return a Cps
-def test_with_mercury_cps(sources, queues, parallelism: 1, &block)
+def test_with_mercury_cps(sources, queues, **kws, &block)
```

Comment:
The same default is now provided in `Mercury::new` after multiple fallthroughs. I'm not sure why I singled out parallelism here before.